gitlab/resource_gitlab_pipeline_schedule_variable_test.go

```diff
@@ -54,6 +54,48 @@ func TestAccGitlabPipelineScheduleVariable_basic(t *testing.T) {
 	})
 }
 
+func TestAccGitlabPipelineScheduleVariable_import(t *testing.T) {
+	rInt := acctest.RandInt()
+	resourceName := "gitlab_pipeline_schedule_variable.schedule_var"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckGitlabPipelineScheduleVariableDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccGitlabPipelineScheduleVariableConfig(rInt),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateIdFunc: getPipelineScheduleVariableID(resourceName),
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func getPipelineScheduleVariableID(n string) resource.ImportStateIdFunc {
+	return func(s *terraform.State) (string, error) {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return "", fmt.Errorf("not found: %s", n)
+		}
+
+		pipelineScheduleVariableID := rs.Primary.ID
+		if pipelineScheduleVariableID == "" {
+			return "", fmt.Errorf("no pipeline schedule variable ID is set")
+		}
+		projectID := rs.Primary.Attributes["project"]
+		if projectID == "" {
+			return "", fmt.Errorf("no project ID is set")
+		}
+
+		return fmt.Sprintf("%s:%s", projectID, pipelineScheduleVariableID), nil
```

Comment:
I'm confused how this test is passing. This ID format doesn't match the required format `project_id:pipeline_schedule_id:key`. 😖